src/lib.rs

```diff
@@ -294,15 +294,9 @@ mod tests {
         for _ in 0..(LENGTH + 1) {
             v.push(rand::random::<T>());
         }
-        let v_len = v.len();
         let packed_v = PackedVec::new(v.clone());
-        assert_eq!(packed_v.len(), v_len);
-        let mut iter = packed_v.iter();
-        for number in v {
-            let value: Option<T> = iter.next();
-            assert_eq!(value, Some(number));
-        }
-        assert_eq!(iter.next(), None);
+        assert_eq!(packed_v.len(), v.len());
+        packed_v.iter().zip(v.iter()).all(|(x, y)| x == *y);
```

Comment:
Doesn't this need to be inside an `assert`?

Reply:
It does, I must've hit undo before committing... My bad!